packages/flutter/lib/src/widgets/text.dart

```diff
@@ -164,6 +164,11 @@ class DefaultTextStyle extends InheritedWidget {
 /// for example, to make the text bold while using the default font family and
 /// size.
 ///
+/// The [Text] widget can also be created directly from a [TextSpan]. Default text
+/// styles will not be provided to the span in this case, but they are used as default
```

Comment:
i don't understand what this line means

Reply:
Let me try again

Reply:
I combined this doc comment with the next line, since users no longer need to always use RichText.  RichText is still mentioned at the bottom, but I could add another reference to it earlier.